<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-QS3L28C" failures="1" tests="6" name="Default test" time="0.155" errors="0" timestamp="2022-09-18T15:53:15 IST">
  <testcase classname="tests_extendR.ExecuteTests_PF" name="@AfterMethod screenshot" time="0.008">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests_extendR.ExecuteTests_PF.screenshot(ExecuteTests_PF.java:98)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:695)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:671)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:655)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod screenshot -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="@AfterMethod screenshot" time="-1.663496595544E9">
    <skipped/>
  </testcase> <!-- @AfterMethod screenshot -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="@AfterMethod screenshot" time="-1.663496595552E9">
    <skipped/>
  </testcase> <!-- @AfterMethod screenshot -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="@AfterMethod screenshot" time="-1.663496595573E9">
    <skipped/>
  </testcase> <!-- @AfterMethod screenshot -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="@AfterClass endClass" time="-1.663496595577E9">
    <skipped/>
  </testcase> <!-- @AfterClass endClass -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="Test_OpenWebsite" time="0.013">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests_extendR.ExecuteTests_PF.Test_OpenWebsite(ExecuteTests_PF.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Test_OpenWebsite -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="Test_Username" time="0.0">
    <skipped/>
  </testcase> <!-- Test_Username -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="Test_Password" time="0.0">
    <skipped/>
  </testcase> <!-- Test_Password -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="Validate_Username" time="0.0">
    <skipped/>
  </testcase> <!-- Validate_Username -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="Validate_Password" time="0.0">
    <skipped/>
  </testcase> <!-- Validate_Password -->
  <testcase classname="tests_extendR.ExecuteTests_PF" name="TestLogin_ValidInput" time="0.0">
    <skipped/>
  </testcase> <!-- TestLogin_ValidInput -->
</testsuite> <!-- Default test -->
